### PR TITLE
add All Transactions Tab

### PR DESCRIPTION
**Pull request by ippudo**

### CODE REVIEW

**Comment by ippudo** [comment]
#1018 
#1029

File: src/scenes/Event/components/HistoryTable/index.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ import { ResultHistory, TransactionHistory } from '../';
 
 const { TOPIC } = EventType;
 const TAB_RESULTHISTORY = 0;
-const TAB_TRANSACTIONHISTORY = 1;
+const TAB_MYTRANSACTIONHISTORY = 1;
+const TAB_ALLTRANSACTIONHISTORY = 2;
```
Comment (dwalintukan) [inline src/scenes/Event/components/HistoryTable/index.js]: change to `TAB_RESULT_HISTORY, TAB_MY_TRANSACTIONS, TAB_ALL_TRANSACTIONS`

File: src/scenes/Event/components/HistoryTable/index.js [hunk 2]
```diff
@@ -58,14 +59,20 @@ export default class HistoryTable extends Component {
...
+          {this.props.transactionHistory && <Tab
+            value={TAB_ALLTRANSACTIONHISTORY}
+            label={intl.formatMessage({ id: 'str.allTransactions', defaultMessage: 'All Transactions' })}
+            className={classes.pickerTab}
+          />}
         </Tabs>
         <div className={classes.subTableContainer}>
           {tabValue === TAB_RESULTHISTORY && <ResultHistory oracles={eventPage.oracles} currentEvent={currentEvent} />}
-          {tabValue === TAB_TRANSACTIONHISTORY && <TransactionHistory options={currentEvent.options} />}
+          {tabValue === TAB_MYTRANSACTIONHISTORY && <TransactionHistory my options={currentEvent.options} />}
```
Comment (dwalintukan) [inline src/scenes/Event/components/HistoryTable/index.js]: change `my` => `myTransactions`

File: src/scenes/Event/components/TransactionHistory/index.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ export default class TransactionHistory extends Component {
             </TableHead>
             <TableBody>
               {transactions.map((transaction) => (
-                <TxRow key={transaction.txid} transaction={transaction} />
+                (!(this.props.my && transaction.senderAddress !== wallet.currentAddress)) && <TxRow key={transaction.txid} transaction={transaction} />
```
Comment (dwalintukan) [inline src/scenes/Event/components/TransactionHistory/index.js]: this doesn't seem like it will show the All Transactions list?

Comment (ippudo) [reply src/scenes/Event/components/TransactionHistory/index.js]: If param `this.props.my` set to false, the transaction will not filtered.
`<TransactionHistory my />` will show the filtered transactions and `<TransactionHistory />` will show all transactions.

Comment (dwalintukan) [reply src/scenes/Event/components/TransactionHistory/index.js]: got it, you were only applying filtering for MyTransactions

**Pull request merged**